install-and-configure/install/etl-backup/query-service-replicas.md

```diff
@@ -4,26 +4,26 @@
 This feature is only supported on Kubecost Enterprise plans.
 {% endhint %}
 
-The query service replica (QSR) is a standalone query service that executes independently of Kubecost's metric emission and data model and is able to access Kubecost data. It allows for improved horizontal scaling by being able to handle queries for larger intervals, and multiple simultaneous queries.
+The query service replica (QSR) is a scale-out query service that reduces load on the cost-model pod. It allows for improved horizontal scaling by being able to handle queries for larger intervals, and multiple simultaneous queries.
 
 ## Overview
 
-The query service will forward `/model/allocation` and `/model/assets` requests to a Deployment of Query Services and managed by a Load Balancer.
+The query service will forward `/model/allocation` and `/model/assets` requests to the Query Services StatefulSet.
 
 The diagram below demonstrates the backing architecture of this query service and its functionality.
 
 <figure><img src="../../../.gitbook/assets/image (5).png" alt=""><figcaption></figcaption></figure>
 
 ## Prerequisites
 
-In order to make use of QSRs, you must first have enabled ETL backups. As of v1.100+ of Kubecost, this is enabled by default if you enable Thanos. To learn more about ETL backups, see the [ETL Backup](https://docs.kubecost.com/install-and-configure/install/etl-backup) doc.
+There are two methods to implement QSR. For environments that have Kubecost [Federated ETL](https://docs.kubecost.com/install-and-configure/install/multi-cluster/federated-etl) enabled, the Federated ETL object-store must be used. Federated ETL is, itself, a backup mechanism.
 
-## Enabling QSR
+For all other environments, QSR will target the ETL backups. As of Kubecost v1.100+, this is enabled by default if you enable Thanos. To learn more about ETL backups, see the [ETL Backup](https://docs.kubecost.com/install-and-configure/install/etl-backup) doc.
```

Comment:
Suggest calling out ETL backup. I am reading the original sentence as QSR being enabled by default in Thanos environments.

```
As of Kubecost v1.100+, ETL Backup is enabled by default if you enable Thanos.
```

Reply:
I'm not sure I understand the ask. This sentence is in the original and updated doc. No?

Reply:
can you review again when you have a few? I updated the section in my last commit.